messages/en.json

```diff
@@ -61,6 +60,99 @@
     "readTime": "{count, plural, =0 {less than a minute} one {# minute} other {# minutes}} read",
     "views": "Views"
   },
+  "about": {
+    "whatIsHackerspace": "What is Hackerspace?",
+    "inforAboutHackerSpace": "<p1> Hackerspace NTNU is a student-driven project open to all students, regardless of study direction or Hackerspace membership. We offer a creative arena where students from different disciplines can get help realizing their ideas in an engaged and inclusive environment. Here, you will find new technology at your disposal, including drones, 3D printers, and Virtual Reality equipment. </p1> <p2> Whether you are a first-year student needing help with your first Arduino project or a fourth-year student wanting to create a 3D model of Trondheim, we can provide both equipment and expertise. We also regularly hold courses for both beginners and advanced students in many exciting subjects. </p2> <p3> Come by to see what we're doing and have a chat. </p3> You can find us on the second floor of the A-block in the Realfagbygget at NTNU Gløshaugen.",
+    "showMaps": "Show map!",
+    "FAQ": "Frequently Asked Questions",
+    "canIUseThe3dPriner?": "Can I use the 3D printer?",
+    "answerCanIUseThe3dPriner": "<p1> All students can use the 3D printers and the rest of the workshop, regardless of whether they are members of Hackerspace or not. We have a reservation system available for all students with FEIDE login from the website's navigation bar. </p1> <p2> Come by during opening hours and get help from one of our members.</p2>",
```

Comment:
Go through all the messages below and simplify them. They should be camelCase and not contain special characters